js/main.js

```diff
@@ -0,0 +1,43 @@
+$('#Read_more').click(function () {
```

Comment:
Sugeruję osobne HTML'e dla każdej podstrony(jak coś w przyszłości ulegnie zmianie to mniejsza szansa, że strona się wywali tzw. regresja). Jeżeli już bardzo tak chcemy podążać drogą jQuery to będzie bałagan, ale taki tip: można zaznaczać kilka class za pomocą jQuery w formacie $('#klasa1,#klasa2,#klasa2) - https://api.jquery.com/multiple-selector/